packages/react/src/SearchControl/SearchControl.tsx

```diff
@@ -80,7 +81,8 @@ export interface SearchControlProps {
   onClick?: (e: SearchClickEvent) => void;
   onAuxClick?: (e: SearchClickEvent) => void;
   onNew?: () => void;
-  onExport?: () => void;
+  onExportCSV?: () => void;
```

Comment:
Can we rename this to `onExportCsv`?  By convention, we always use `camelCase`, even with abbreviations like `Id`, etc.